src/constitutive_relations/column_integrators/water_table_depth_evaluator.cc

```diff
@@ -32,22 +36,32 @@ ParserWaterTableDepth::ParserWaterTableDepth(Teuchos::ParameterList& plist, cons
 
 IntegratorWaterTableDepth::IntegratorWaterTableDepth(Teuchos::ParameterList& plist,
                                                      std::vector<const Epetra_MultiVector*>& deps,
-                                                     const AmanziMesh::Mesh* mesh)
+                                                     const AmanziMesh::Mesh* mesh) : mesh_(mesh)
 {
-  AMANZI_ASSERT(deps.size() == 3);
+  AMANZI_ASSERT(deps.size() == 4);
   sat_ = deps[0];
-  cv_ = deps[1];
-  surf_cv_ = deps[2];
+  pres_ = deps[1];
+  cv_ = deps[2];
+  surf_cv_ = deps[3];
+  is_interp_ = plist.get<bool>("determined by pressure interpolation", false);
```

Comment:
Usually I try to make parameter names in "imperative mood", can we make this:  "interpolate depth from pressure".

Also, is this parameter in the documentation for this file?  Please add if not.